app/routes/my-projects/to-review.js

```diff
@@ -7,4 +7,13 @@ export default class MyProjectsToReviewRoute extends Route {
 
   @service
   store;
+
+  async model() {
+    return this.store.query('assignment', {
+      tab: 'to-review',
+      include: 'project.milestones,project.dispositions,project.actions,dispositions,dispositions.action',
+    }, {
+      reload: true,
+    });
+  }
```

Comment:
For demo tomorrow we need to find a quick fix around this... hard to explain over comments

Reply:
Okay let's chat tomorrow